src/includes/getting-started-config/javascript.svelte.mdx

```diff
@@ -0,0 +1,55 @@
+To use the SDK, initialize it in your Svelte entry point before bootstrapping your app. In a typical Svelte project, that is your `main.js` or `main.ts` file.
+
+```typescript {filename: main.ts}
+import "./app.css";
+import App from "./App.svelte";
+
+import * as Sentry from "@sentry/svelte";
+import { BrowserTracing } from "@sentry/tracing";
+
+// Initialize the Sentry SDK here
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  release: "my-project-name@2.3.12",
```

Comment:
Do we want to list `release` here? We don't in other JS SDKs I believe. Same thing below.